src/services/token.service.ts

```diff
@@ -19,16 +19,22 @@ export class TokenService implements TokenServiceAbstraction {
     constructor(private storageService: StorageService) {
     }
 
-    setTokens(accessToken: string, refreshToken: string): Promise<any> {
+    async setTokens(accessToken: string, refreshToken: string): Promise<any> {
         return Promise.all([
-            this.setToken(accessToken),
-            this.setRefreshToken(refreshToken),
+            await this.setToken(accessToken),
+            await this.setRefreshToken(refreshToken),
```

Comment:
You don't need to await these. The purpose of `Promise.all` is to run things concurrently. These methods already return a promise.

Reply:
Updated.